pkg/cmd/node/node_diagnostics_delete.go

```diff
@@ -13,6 +13,11 @@ func newCmdNodeDiagnosticsDelete(ctx api.Context) *cobra.Command {
 		Short: "Delete a diagnostics bundle",
 		Args:  cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			err := ctx.Deprecated("This command is deprecated since DC/OS 1.14, please use 'dcos diagnostics delete' instead.")
```

Comment:
Deprecation is for the wrong version but I'm also not sure we'll want to backport the deprecation warning.